third_party/libgcc/udivmoddi4.c

```diff
@@ -23,11 +23,15 @@ a copy of the GCC Runtime Library Exception along with this program;
 see the files COPYING3 and COPYING.RUNTIME respectively.  If not, see
```

Comment:
Following the example set by PR #2605, I believe we also need to update third_party/libgcc/README.dynamorio.

Reply:
Done